Makefile

```diff
@@ -13,6 +13,13 @@ imports = \
 xcodeproj:
 	swift package generate-xcodeproj
 
+bootstrap: common-crypto-mm xcodeproj
+
+common-crypto-mm:
+	-@mkdir "$(COMMON_CRYPTO_PATH)" || true
+	-@touch "$(COMMON_CRYPTO_MODULE_PATH)" || true
```

Comment:
This should never fail, no? And if you're `echo`ing the next line, maybe it's not even necessary?

Reply:
ha... yahhhhh